src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -179,6 +210,91 @@ function removeRubricCol(index, questionNum) {
                                          StatusType.WARNING);
 }
 
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
+    if ($(`#rubricEditTable-${questionNum}`).length === 0
+            && $(`.rubricCol-${questionNum}-${colIndex}`).length === 0
+            && typeof isSwapLeft !== 'boolean') {
+        // question and column should exist, isSwapLeft must be boolean
+        return;
+    }
+
+    const numberOfRows = parseInt($(`#rubricNumRows-${questionNum}`).val(), 10);
+    let swapColIndex;
+    const swapCellAccessorStr = `#rubric-options-row-${questionNum} .rubricCol-${questionNum}-${colIndex}`;
+    const rubricCellSelector = `td[class*='rubricCol-${questionNum}']`;
+
+    if (isSwapLeft) {
+        if ($(swapCellAccessorStr).prev(rubricCellSelector).length !== 0) {
+            swapColIndex = $(swapCellAccessorStr).prev(rubricCellSelector).attr('data-col');
+        } else {
+            // trying to swap left most column to left
+            return;
+        }
+    } else if ($(swapCellAccessorStr).next(rubricCellSelector).length !== 0) {
+        swapColIndex = $(swapCellAccessorStr).next(rubricCellSelector).attr('data-col');
+    } else {
+        // trying to swap right most column to right
+        return;
+    }
```

Comment:
I was going to suggest rewriting this so it would look like

```javascript
if (isSwapLeft) {
    // left stuff
} else {
    // right stuff
}
```

but I realised it adds a redundant level of conditionals, so this should be fine.